roles/host-provision/tasks/rhel-config-tasks.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: config-tasks | Validate internet connectivity via control node
```

Comment:
I was intrigued by the differences between this check and proxy.yml's in https://github.com/google/bms-toolkit/pull/89/commits/f457bf39352144f9228a8639e6fd07e87094f885#diff-0df9002f4dbec9645fbab32b593aaebcce4a4556fd709f3744e09897d3f2c119R27 ?  Notably, we set proxy env variables there but not here.  I believe this works as a side effect of ansible's non-persistent SSH connections:  if rhel-config-tasks.yml is run in a separate shell, it presumably runs the newly-created /etc/profile.d/proxy.sh which sets the env variables.  A config like this would break if using ansible's local connection plugin, but I presume host-provision doesn't let you do this?

Reply:
Right, I didn't include the `env` variable due to the reasoning (`/etc/profile`).
In hind sight, if a user were to choose `--proxy-setup false` on a reimaged host, it will break.

Adding the env back.

Reply:
As discussed, removing the proxy/internet connectivity check from here, as the message itself is self-sufficient:
```
[root@BMS-HOST-1 yum.repos.d]# /sbin/subscription-manager register --force --auto-attach --username VALUE_SPECIFIED_IN_NO_LOG_PARAMETER --password asdf
Unregistering from: subscription.rhsm.redhat.com:443/subscription
Network error. Please check the connection details, or see /var/log/rhsm/rhsm.log for more information.
```